api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -50,6 +50,8 @@
     "CategoryEnum", {subcategory_id: subcategory_id for subcategory_id in subcategories.EVENT_SUBCATEGORIES}
 )
 
+offer_price_model = pydantic_v1.conint(strict=True, ge=0, lt=30000)  # 300 euros
```

Comment:
```suggestion
if typing.TYPE_CHECKING:
    offer_price_model = pydantic_v1.StrictInt
else:
    offer_price_model = pydantic_v1.conint(strict=True, ge=0, lt=30000)  # 300 euros
```
te permet de te débarasser de ces type ignore